core/drivers/ls_dspi.c

```diff
@@ -378,6 +378,18 @@ static void ls_dspi_end(struct spi_chip *chip)
 	io_setbits32(data->base + DSPI_MCR, DSPI_MCR_HALT);
 }
 
+/*
+ * Flush DSPI module
+ * chip:	spi_chip instance
+ */
+static void ls_dspi_flush(struct spi_chip *chip)
+{
+	struct ls_dspi_data *data = container_of(chip, struct ls_dspi_data,
+						  chip);
+
+	dspi_flush_fifo(data);
```

Comment:
Make `dspi_flush_fifo `as static also as it gets exposed via `ls_dspi_ops`.
Also remove it from `ls_dspi.h`